heltour/tournament/models.py

```diff
@@ -77,9 +77,11 @@ class Meta:
     ('blue', 'Blue'),
     ('green', 'Green'),
     ('red', 'Red'),
+    ('yellow', 'Yellow'),
 )
 RATING_TYPE_OPTIONS = (
     ('classical', 'Classical'),
+    ('rapid', 'Rapid'),
```

Comment:
Should also add "chess960" since this is the `perf` value, not the `speed` value.

Reply:
I added chess960 here, but as per the convo in the dev channel, we'll be using Rapid for this as the lack of time-control specific ratings really messes with it, so the chess960 mods feel that Rapid is a better indication of playing strength than the chess960 rating.